.github/workflows/docs.yml

```
@@ -0,0 +1,64 @@
+name: Deploy
+
+on:
+    push:
+        tags:
+            - "v*"
+    workflow_dispatch:
+
+jobs:
+    docs:
+        name: Deploy
+        runs-on: ubuntu-latest
+        env:
+            DST_REMOTE: origin
+            DST_BRANCH: test-pages
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+            - name: Configure git
+              run: |
+                  git config --global user.email "${DOCS_DEPLOY_EMAIL}"
+                  git config --global user.name "${DOCS_DEPLOY_NAME}"
+            - name: Setup Node.js
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20.x
+            - name: Determine version
+              run: echo "VERSION=v$(npm pkg get version | jq -r)" | tee -a $GITHUB_ENV
+            - name: Install dependencies
+              run: npm ci
+            - name: Build (minimal)
+              run: npm exec lerna -- run --include-dependencies --scope=@fkui/vue --scope=@fkui/vue-labs build
+            - name: Build documentation
+              run: npm run build:docs
+              env:
+                  DOCS_MOTD_URL: "/latest"
+                  DOCS_SOURCE_URL_FORMAT: "https://github.com/${{ github.repository }}/tree/main/{{ path }}"
```

```suggestion
                  DOCS_SOURCE_URL_FORMAT: "https://github.com/${{ github.repository }}/blob/{{ commit }}/{{ path }}"
```

Antingen om `docs-generator` har stöd för det eller om vi läser ut det i pipeline.

Tanken bakom detta är att varje release länkar till källkoden som den var då, inte till senaste i master. Dels för att det är rimligt men också för att undvika brutna länkar när vi refaktoriserar och flyttar kod/filer.